frontend/public/components/masthead-toolbar.jsx

```diff
@@ -12,6 +12,14 @@ import { AboutModal } from './about-modal';
 import { getAvailableClusterUpdates, clusterVersionReference } from '../module/k8s/cluster-settings';
 import { commandLineToolsModal } from './modals';
 
+const SystemStatusButton = ({statuspageData, onClick, className}) => !_.isEmpty(_.get(statuspageData, ['incidents']))
+  ? <ToolbarItem className={className}>
+    <Button variant="plain" aria-label="System Status" onClick={onClick}>
```

Comment:
Make this a real link instead of a button? That's better semantically and then you don't need to pass in the `onClick` handler.